Sources/BraintreeCard/CreditCardPOSTBody.swift

```diff
@@ -0,0 +1,139 @@
+// swiftlint:disable nesting
+
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+// swiftlint:disable nesting
+struct CreditCardPOSTBody: Encodable {
```

Comment:
Can we add a docstring for this? Can use `/// The POST body for <endpoint_here>` like we have in other files.